pkg/framework/resource/resource.go

```diff
@@ -76,6 +75,15 @@ func New(config ResourceConfig) (*Resource, error) {
 	return c, nil
 }
 
+func (r *Resource) DeleteResource(name string) error {
```

Comment:
Within the `resource` package on something that is created by `New` the method should rather be called `Delete`. Same with `Install` below.

Reply:
100% agreed, changed.